### DISCUSSION TITLE
Upgrade from Free Plan to Pro Lite Plan.

### DISCUSSION THREAD

**Discussion by prashanthcoe19** [FortAwesome/Font-Awesome#20346] [Q&A] [answered]
I have created a kit in fontawesome. It is a free kit and it has reached its 10000 limit.
Now, If I upgrade my plan to pro lite, will I be able to use the same kit or do I need to create a new kit?

**Comment by sensibleworld** [comment, marked as answer]
The same kit will continue to work after upgrading. Though, if you want to use Pro icons in that kit, you'll need to change it to a "Pro" kit from the kit settings page. (Embed code will remain the same even after that.)
